src/baguetteBox.js

```diff
@@ -313,8 +327,19 @@
         if (options.buttons === 'auto' && ('ontouchstart' in window || imagesMap[currentGallery].length === 1)) {
             options.buttons = false;
         }
-        // Set buttons style to hide or display them
-        previousButton.style.display = nextButton.style.display = (options.buttons ? '' : 'none');
+
+        if (options.buttons === false) {
+            previousButtonContent.setAttribute('aria-hidden', 'true');
+            nextButtonContent.setAttribute('aria-hidden', 'true');
+
+            // Set buttons style to hide
+            previousButton.style.display = nextButton.style.display = 'none';
+        } else {
+            previousButtonContent.setAttribute('aria-hidden', 'false');
+            nextButtonContent.setAttribute('aria-hidden', 'false');
+            closeButtonContent.setAttribute('aria-hidden', 'false');
+        }
```

Comment:
Weird, I expected HTML_Codesniffer to not be so off :/

Cool, then I'll rebase this just to change the `X` to `&times;`.

Thanks for the help!